.gitattributes

```diff
@@ -0,0 +1,4 @@
+server/manifest.go -diff -merge
```

Comment:
I'm not convinced that this line is needed. These files aren't binary files and can be easily checked. Why do opted to add these line?

Reply:
My idea with those lines was:

- avoid to show the diff in the cli to reduce the noise
- treat it as a binary, because is a generated file, so we can avoid merge conflicts

I thought that it could be useful but if ypu think is not I can remove then :)

Reply:
I'm 1/5 that I actually want to see the diff on the cli. This allows me to double check that the version bump was made correctly.

I see your point on 2., but I suppose there are rarely merge conflicts and if so I want to resolve them myself.

@lieut-data Thoughts on the discussion?

Reply:
I'm 0/5 right now. I agree that it might be "too magical" to hide in the CLI.

Reply:
@ethervoid Would you be fine with removing these lines?

Reply:
Sure! Done :)